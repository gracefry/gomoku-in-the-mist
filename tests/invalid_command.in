Who
who
who 
Place A1
placeA1
place  A1
place A1
who
place A1 
random
term
