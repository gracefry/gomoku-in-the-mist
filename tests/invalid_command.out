Invalid!
B
Invalid!
Invalid!
Invalid!
Invalid!
W
Invalid!
Invalid!
